### ISSUE TITLE
Mannequin, Armor Stand lacking interaction parity

### ISSUE THREAD

**Issue by asleeves** [anegostudios/VintageStory-Issues#9737]
Armor stands use crouch + interact for both attach and detach behaviors, whereas mannequins use interact for attach and shift + interact for detach. Furthermore, mannequins don't display block interaction help.
